ScienceJournal/Extensions/CMMagneticField+ScienceJournal.swift

```diff
@@ -0,0 +1,42 @@
+/*
+ *  Copyright 2019 Google Inc. All Rights Reserved.
```

Comment:
So the internal presubmit passes.
```suggestion
 *  Copyright 2019 Google LLC. All Rights Reserved.
```

Reply:
Love having commit suggestions! Thanks!